docs/getting-started/developers/index.md

```diff
@@ -10,87 +10,97 @@ This page describes how to get started building an application with Warp compone
 
 If you are migrating from Fabric to Warp, please visit the [Migration page](/migration/developers/).
 
-This is the initial version of Warp and is currently a work in progress. Any elements, components or inline styled/custom CSS that are not implemented using Fabric, is required to be managed in order to make them themeable.
+If you have any questions or need clarification, please don't hesitate to reach out to the Warp team on the #nmp-warp-design-system channel on Slack!
+
+Warp is still a **work in progress**. We kindly request that you **hold off on merging your Warp migration pull requests to your main branch for now**. Some breaking changes are expected during the summer, and it would be great to have this completed before teams begin merging to the main branch. Additionally, we want to thoroughly assess that Warp components and classes looks and behaves as expected, before granting teams the green light. We're actively working on getting Warp ready for Finn production.
+
+For the Big Launch, our primary focus with Warp is to support the existing styling and components that were available in Fabric. Adding new features to Warp is not our current priority. However, we encourage you to share your ideas for future features in our channel on Slack! We will add them to our backlog for consideration.
```

Comment:
Should I remove this part? 🤔 

However, we encourage you to share your ideas for future features in our channel on Slack! We will add them to our backlog for consideration.

Reply:
I think we can remove this part because it seems to be covered in this section: https://warp-ds.github.io/tech-docs/collaborate/contribute/#requesting-changes-to-an-existing-component

Reply:
> We kindly request that you **hold off on merging your Warp migration pull requests to your main branch for now**.

Maybe we could emphasise that we would like to hold off the deployment to production of Finn apps which migrated to Warp?